Slot ThreadPool [14767] (#241)
eProsima/DDS-Router: docs/rst/notes/forthcoming_version.rst

  DDS Router.
  This slightly improves the **performance** of the router.

* New :code:`ThreadPool` class for handling parallel processing.

>   This allow to limit the number of threads spawned by the DDS Router, improving the performance of the application.

Reviewer: ```suggestion
  Limit the number of threads spawned by the DDS Router, improving the performance of the application.
```

>   This could be configured by the user using the YAML configuration :ref:`thread_configuration`.

Reviewer: ```suggestion
  The user can set-up this parameter on the YAML configuration :ref:`thread_configuration`.
```
